src/index.js

```diff
@@ -98,16 +98,22 @@ class FlipMotion extends Component<void, Props, State> {
     }
   }
   render() {
-    const { style, childStyle } = this.props
+    const style = this.props.style
+    const childStyle = this.props.childStyle
+    const Element = this.props.element || "div"
+    const ChildElement = this.props.childElement || "div"
     return (
       <TransitionMotion
         styles={this.getStyles()}
         willEnter={this.willEnter}
       >
         {(styles) =>
-          <div style={style}>
+          <Element
+            style={style}
+            className={this.props.className}
+          >
             {styles.map((item) =>
-              <div
+              <ChildElement
                 key={item.key}
                 style={item.style && {
                   ...childStyle,
```

Comment:
I think we need to warn if there's any colliding value (e.g. `childStyle.hasOwnProperty("transform")`) or find a safe merge strategy.